swagger.yaml

```diff
@@ -630,7 +630,7 @@ paths:
                 items:
                   $ref: "#/components/schemas/Error"
 
-  "/questionnaire/{id}":
+  "/questionnaires/{id}":
```

Comment:
All questionnaire endpoints should be renamed to questionnaires, including post put and delete